probes/toolDependabotInstalled/impl.go

```diff
@@ -0,0 +1,41 @@
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package toolDependabotInstalled
```

Comment:
do all these probes need their own package?

Reply:
good question. My motivations were:
1. Avoid collision of variables. As the number of probes grow, collision will happen more. A dedicated package keeps each probe isolated from one another
2. Wanted to avoid having 50+ files in the same directory

Apart from these, no strong opinion. Happy to change it. I can also re-visit later without blocking other checks' integration.  Let me know.